types/ComponentProductType.json

```
@@ -0,0 +1,20 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "/types/ComponentProductType",
+    "description": "Abstract class that represents a mix of products or single product.",
+    "type": "object",
+
+    "properties": {
+
+        "mixSequence":{
+            "description":"the order of this product in the mix",
+            "type":  "number"
+        },
+        "percent":{
+            "type":"number"
```

Add a description: "The proportion that this product composes of the mix. Can be calculated from Wt/Vol or other measures."

```suggestion
            "type": "number",
            "description": "The proportion that this product composes of the mix. Can be calculated from Wt/Vol or other measures."
```